test/service/test_manifest.py

```diff
@@ -2167,7 +2167,7 @@ def _canned_entities(self):
         def hash_entities(entities: dict[EntityReference, JSON]) -> dict[str, JSON]:
             return {
                 json_hash(contents).digest(): {
-                    'type': ref.entity_type,
+                    'type': 'duos_dataset_registration' if 'duos_id' in contents else ref.entity_type,
```

Comment:
Is there any way we can avoid this? What would the value be if this conditional didn't exist?

Reply:
Agreed at PL to represent the expected contents of the JSONL manifest as a separate canned file, instead of constructing it dynamically from the canned bundles.